site/content/community/02-roadmap.md

```diff
@@ -1,16 +1,17 @@
 ---
-title: "Road Map"
+title: "Project Roadmap"
 weight: 20
-url: community/road-map
+url: community/roadmap
 ---
 
-## Project Road Map
+## Project Roadmap Themes
```

Comment:
I'm suggesting that we use the following content for this page (it's a bit big for using the GitHub "suggestion" feature!). I'd like to make it clearer that we are thinking about the quickstart at the moment. 

## Project Roadmap Themes

### Cluster quickstart with GitOps

It should be easy to create a cluster with various applications pre-installed, e.g. Weave Flux, Helm 2 (Tiller), ALB Ingress controller. It should also be easy to manage these applications in a declarative way using config files in a git repo (with [GitOps](https://www.weave.works/blog/what-is-gitops-really)).

### Declarative configuration management for clusters

One should be able to make EKS cluster configuration through declarative config files (`eksctl apply`). Additionally, they should be able to manage a cluster via a git repo.

### Cluster addons

Understanding how the [add-ons spec by SIG Cluster Lifecycle](https://github.com/kubernetes/enhancements/pull/746) will evolve and how we can implement management of some cluster applications via the addon spec.

Reply:
Sounds good, I'll open another PR, and try to address #977 also.

Reply:
I've incorporated this in #1009, PTAL :)